test/integration/source/default.rb

```diff
@@ -6,4 +6,8 @@
   describe command('npm -v') do
     its('exit_status') { should eq 0 }
   end
+
+  describe file '/home/random/staging' do
+    it { should exist }
```

Comment:
@MarkGibbons and here is the other half of the test (which you can see in multiple test files), asserting that the file exists, thus ensuring that `$NODE_ENV` was successfully passed to `npm_package`.